components/blocks/CallToAction.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <section
+    class="flex flex-col justify-center items-center relative py-20 bg-primary-dark"
+  >
+    <BaseImage
+      class="object-cover h-full w-auto absolute opacity-10 z-0 md:w-full md:h-max"
+      :src="block.background_image.filename"
+      :alt="block.background_image.alt"
+    />
```

Comment:
Now that we're positioned absolute with object-cover, you can remove any h- or w- related classes